llvm/docs/ReleaseNotes.md

```diff
@@ -350,10 +350,16 @@ Changes to the Debug Info
 Changes to the LLVM tools
 ---------------------------------
 
+* llvm-objcopy now supports the following options for MachO:
```

Comment:
I think technically the spelling is "Mach-O". We've not done a great job in the command guide of being consistent, so fixes there welcome too (in a separate commit).

Reply:
This was the only occurrence in ReleaseNotes, so I fixed it in a fixup commit.